onnxruntime/test/optimizer/graph_transform_test.cc

```diff
@@ -312,6 +314,44 @@ TEST(GraphTransformationTests, FuseConvActivation) {
     ASSERT_TRUE(op_to_count[model.second] == 0);
   }
 }
+
+TEST(GraphTransformationTests, FuseConvClip11Activation) {
```

Comment:
low-priority: can we use the same test method as for FuseConvClip11Activation and just pass both models in there.
I think also the convention is that the optimizers should work with the latest opset. Of course it's better if we have multiple versions tested. Maybe also rename the old model to conv_clip6? in case we keep it, that is.

Reply:
Not really. The models test different aspects of fusion so it's not a 1:1 in terms of number of ops, and expected outcomes.